Maestro/subscriptions.json

```diff
@@ -1021,6 +1021,34 @@
         }
       }
     },
+    // Update dependencies in uwp-setup rel/6.1
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt",
```

Comment:
We do have a `release/uwp6.1` servicing branch in BuildTools that it seems like this matches up to. It would be good to switch for consistency and ability to take servicing-specific BuildTools fixes in the future. (Also in uwp-setup's `dependencies.props`.) I'm not sure if uwp-setup depends on anything that came into `master` after `release/uwp6.1` forked, though.